source/features/show-whitespaces.tsx

```diff
@@ -0,0 +1,96 @@
+import './show-whitespaces.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import onPrFileLoad from '../libs/on-pr-file-load';
+import onNewComments from '../libs/on-new-comments';
+import getTextNodes from '../libs/get-text-nodes';
+
+function showWhiteSpacesOn(line: Element): void {
+	const textNodes = getTextNodes(line);
+
+	for (const textNode of textNodes) {
+		const nodeValue = textNode.textContent!;
+		if (nodeValue.length === 0) {
```

Comment:
I'm looking for way to speed up parsing. Can this skip nodes without whitespace? **Perhaps** `.contains` is faster than `/[ \t]/.test(nodeValue)`

```suggestion
		if (nodeValue.length === 0 || !(nodeValue.contains(' ') || nodeValue.contains('\t'))) {
```